toppra/dracula/three_segment_spline.py

```diff
@@ -0,0 +1,154 @@
+from __future__ import print_function
+
+from numpy.testing import assert_almost_equal
+
+import copy
+import inspect
+import math
+import os
+
+import numpy as np
+import matplotlib.pyplot as plt
+import argparse
+import logging
+logging.basicConfig()
```

Comment:
Does this do some magic behind the scenes? Otherwise, it isn't used anwhere

Reply:
removed